### PR TITLE
feat(*): add helm chart

### PR DESCRIPTION
**Pull request by prydonius**

### CODE REVIEW

**Comment by migmartri** [comment]
A couple of things that I'd add to this PR

- [ ] Rebase master
- [ ] build/deploy scripts
- [ ] Be sure that the api endpoint configuration in the client is modified only for the production environment so it does not break the dev env.

File: helm/monocular/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```
Comment (migmartri) [inline helm/monocular/Chart.yaml]: Can you update the description ;)

Take something from the readme.

File: helm/monocular/values.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  replicaCount: 2
+  image:
+    repository: gcr.io/helm-ui/monocular-api
+    tag: latest
+    pullPolicy: Always
+  service:
+    name: monocular-api
+    type: NodePort
+    externalPort: 80
+    internalPort: 8081
```
Comment (migmartri) [inline helm/monocular/values.yaml]: You should be able to run the server with port 80 with the `PORT` env variable.

In any case, I do not think that it is needed.

File: src/ui/rootfs/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+FROM bitnami/nginx
+
+COPY dist/ /app
```
Comment (migmartri) [inline src/ui/rootfs/Dockerfile]: can you add a comment with the process that needs to be previously done?

`ng build --prod ....`

File: helm/monocular/values.yaml (hunk 2 above)
Comment (prydonius) [inline helm/monocular/values.yaml]: I would rather have the container run a non-privileged port, even though it starts as root currently.

**Pull request merged**